packages/connect/src/interceptor.ts

```diff
@@ -140,7 +140,7 @@ export interface StreamResponse<
   readonly message: AsyncIterable<MessageShape<O>>;
 }
 
-interface RequestCommon<I extends DescMessage, O extends DescMessage> {
+export interface RequestCommon<I extends DescMessage, O extends DescMessage> {
```

Comment:
Nice clean up.

Can you mark the two new exports as `@private`? They are not exported in the package, but this makes it unmistakably clear that they shouldn't be.